include/alpaka/mem/view/ViewSubView.hpp

```diff
@@ -53,23 +53,23 @@ namespace alpaka
                 "The dev type of TView and the Dev template parameter have to be identical!");
 
             static_assert(
-                std::is_same_v<TIdx, Idx<View>>,
+                std::is_same_v<TIdx, alpaka::Idx<View>>,
```

Comment:
@ichinii Is this change required because you got the same issue as in `BufUniformCudaHipRt.hpp` or do you changed it because it is equal to the assert in the `BufUniformCudaHipRt.hpp`. 

I am asking because the include for the Idx trait is in this file available and this file was not shown in the error message you posted.

Reply:
It is the same issue

```
[build] C:\Users\ich\Desktop\hzb\forkalpaka\include\alpaka/mem/view/ViewSubView.hpp(56): error : Idx is not a template [C:\Users\ich\Desktop\hzb\forkalpaka\build\gpu-cuda-nvcc\test\unit\mem\copy\bufSlicingTest.vcxproj]
[build]                     std::is_same_v<TIdx, Idx<View>>,
[build]                                          ^
[build]             detected during:
[build]               instantiation of "alpaka::ViewSubView<TDev, TElem, TDim, TIdx>::ViewSubView(TQualifiedView &, const TExtent &, const TOffsets &) [with TDev=alpaka::DevCpu, TElem=int32_t, TDim=std::integral_constant<size_t, 1ULL>, TIdx=int64_t, TQualifiedView=alpaka::BufCpu<int32_t, std::integral_constant<size_t, 1ULL>, int64_t>, TOffsets=alpaka::Vec<std::integral_constant<size_t, 1ULL>, int64_t>, TExtent=alpaka::Vec<std::integral_constant<size_t, 1ULL>, int64_t>]" at line 76 of C:\Users\ich\Desktop\hzb\forkalpaka\test\unit\mem\copy\src\BufSlicing.cpp
[build]               instantiation of "auto TestContainer<TDim, TIdx, TAcc, TData, Vec>::copySliceOnDevice(TestContainer<TDim, TIdx, TAcc, TData, Vec>::BufDevice, Vec, Vec)->TestContainer<TDim, TIdx, TAcc, TData, Vec>::BufDevice [with TDim=std::integral_constant<size_t, 1ULL>, TIdx=int64_t, TAcc=alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int64_t>, TData=int32_t, Vec=alpaka::Vec<std::integral_constant<size_t, 1ULL>, int64_t>]" at line 138 of C:\Users\ich\Desktop\hzb\forkalpaka\test\unit\mem\copy\src\BufSlicing.cpp
[build]               instantiation of "void CATCH2_INTERNAL_TEMPLATE_TEST_5<TestType>() [with TestType=std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int64_t>, int32_t>]" at line 109 of C:\Users\ich\Desktop\hzb\forkalpaka\test\unit\mem\copy\src\BufSlicing.cpp
[build]               instantiation of "void <unnamed>::ns_CATCH2_INTERNAL_TEMPLATE_TEST_4::CATCH2_INTERNAL_TEMPLATE_TEST_4<Types...>::reg_tests() [with Types=<std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int64_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, int64_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, int64_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, int64_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, int64_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, int64_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, uint64_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, uint64_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, uint64_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, uint64_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, uint64_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, uint64_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int32_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, int32_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, int32_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, int32_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, int32_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, int32_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, uint32_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, uint32_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, uint32_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, uint32_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, uint32_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, uint32_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int64_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, int64_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, int64_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, int64_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, int64_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, int64_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, uint64_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, uint64_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, uint64_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, uint64_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, uint64_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, uint64_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int32_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, int32_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, int32_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, int32_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, int32_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, int32_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, uint32_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, uint32_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, uint32_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, uint32_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, uint32_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, uint32_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int64_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, int64_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, int64_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, int64_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, int64_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, int64_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, uint64_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, uint64_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, uint64_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, uint64_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, uint64_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, uint64_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int32_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, int32_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, int32_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, int32_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, int32_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, int32_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, uint32_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, uint32_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, uint32_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, uint32_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, uint32_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, uint32_t>, double>>]" at line 109 of C:\Users\ich\Desktop\hzb\forkalpaka\test\unit\mem\copy\src\BufSlicing.cpp
[build]   
[build] C:\Users\ich\Desktop\hzb\forkalpaka\include\alpaka/mem/view/ViewSubView.hpp(55): error : static assertion failed with "The idx type of TView and the TIdx template parameter have to be identical!" [C:\Users\ich\Desktop\hzb\forkalpaka\build\gpu-cuda-nvcc\test\unit\mem\copy\bufSlicingTest.vcxproj]
[build]                 static_assert(
[build]                 ^
[build]             detected during:
[build]               instantiation of "alpaka::ViewSubView<TDev, TElem, TDim, TIdx>::ViewSubView(TQualifiedView &, const TExtent &, const TOffsets &) [with TDev=alpaka::DevCpu, TElem=int32_t, TDim=std::integral_constant<size_t, 1ULL>, TIdx=int64_t, TQualifiedView=alpaka::BufCpu<int32_t, std::integral_constant<size_t, 1ULL>, int64_t>, TOffsets=alpaka::Vec<std::integral_constant<size_t, 1ULL>, int64_t>, TExtent=alpaka::Vec<std::integral_constant<size_t, 1ULL>, int64_t>]" at line 76 of C:\Users\ich\Desktop\hzb\forkalpaka\test\unit\mem\copy\src\BufSlicing.cpp
[build]               instantiation of "auto TestContainer<TDim, TIdx, TAcc, TData, Vec>::copySliceOnDevice(TestContainer<TDim, TIdx, TAcc, TData, Vec>::BufDevice, Vec, Vec)->TestContainer<TDim, TIdx, TAcc, TData, Vec>::BufDevice [with TDim=std::integral_constant<size_t, 1ULL>, TIdx=int64_t, TAcc=alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int64_t>, TData=int32_t, Vec=alpaka::Vec<std::integral_constant<size_t, 1ULL>, int64_t>]" at line 138 of C:\Users\ich\Desktop\hzb\forkalpaka\test\unit\mem\copy\src\BufSlicing.cpp
[build]               instantiation of "void CATCH2_INTERNAL_TEMPLATE_TEST_5<TestType>() [with TestType=std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int64_t>, int32_t>]" at line 109 of C:\Users\ich\Desktop\hzb\forkalpaka\test\unit\mem\copy\src\BufSlicing.cpp
[build]               instantiation of "void <unnamed>::ns_CATCH2_INTERNAL_TEMPLATE_TEST_4::CATCH2_INTERNAL_TEMPLATE_TEST_4<Types...>::reg_tests() [with Types=<std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int64_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, int64_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, int64_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, int64_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, int64_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, int64_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, uint64_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, uint64_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, uint64_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, uint64_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, uint64_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, uint64_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int32_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, int32_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, int32_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, int32_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, int32_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, int32_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, uint32_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, uint32_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, uint32_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, uint32_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, uint32_t>, int32_t>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, uint32_t>, int32_t>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int64_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, int64_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, int64_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, int64_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, int64_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, int64_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, uint64_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, uint64_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, uint64_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, uint64_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, uint64_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, uint64_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int32_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, int32_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, int32_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, int32_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, int32_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, int32_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, uint32_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, uint32_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, uint32_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, uint32_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, uint32_t>, float>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, uint32_t>, float>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int64_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, int64_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, int64_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, int64_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, int64_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, int64_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, uint64_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, uint64_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, uint64_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, uint64_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, uint64_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, uint64_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, int32_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, int32_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, int32_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, int32_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, int32_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, int32_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 1ULL>, uint32_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 1ULL>, uint32_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 2ULL>, uint32_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 2ULL>, uint32_t>, double>, std::tuple<alpaka::AccCpuSerial<std::integral_constant<size_t, 3ULL>, uint32_t>, double>, std::tuple<alpaka::AccGpuUniformCudaHipRt<alpaka::ApiCudaRt, std::integral_constant<size_t, 3ULL>, uint32_t>, double>>]" at line 109 of C:\Users\ich\Desktop\hzb\forkalpaka\test\unit\mem\copy\src\BufSlicing.cpp
```